subprojects/packagefiles/libxml2/meson.build

```diff
@@ -78,9 +78,29 @@ if math_lib.found()
 endif
 
 if meson.version().version_compare('>= 0.60')
-  iconv_lib = dependency('iconv', required: false)
+  iconv_lib = dependency('iconv', required: get_option('iconv'))
 else
-  iconv_lib = dependency('', required: false)
+  if get_option('iconv').disabled()
+    iconv_lib = dependency('', required: false)
+  else
```

Comment:
I didn't bother doing this because of all the work involved. In any case, I doubt many will be using such an old meson version.